instrumentation/opentelemetry-instrumentation-remoulade/src/opentelemetry/instrumentation/remoulade/utils.py

```diff
@@ -0,0 +1,25 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+def attach_span(span_registry, message_id, span, is_publish=False):
+    span_registry[(message_id, is_publish)] = span
+
+
+def detach_span(span_registry, message_id, is_publish=False):
```

Comment:
Yes it would make sense. However as I stated below, the code is heavily inspired from Celery's instrumentation which does not use two span registries but only one.
I can still implement two span registries if you wish me to.

Reply:
Not a blocking comment. Just would make it so we don't have to keep track of `is_publish`.